homeassistant/components/esphome/__init__.py

```diff
@@ -36,7 +36,7 @@
 import homeassistant.helpers.config_validation as cv
 import homeassistant.helpers.device_registry as dr
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
-from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.entity import DeviceInfo as EntityDeviceInfo, Entity
```

Comment:
I'd rather import third party module as different name to be consistent across the codebase.

Reply:
Done.